tests/Unit/DependencyInjection/Factory/RoutingAdminFactoryTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Tests\Unit\DependencyInjection\Factory;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class RoutingAdminFactoryTest extends AbstractFactoryTest
+{
+    public function testParametersBundle()
+    {
+        $this->container->setParameter('kernel.bundles', [
+            'CmfRoutingBundle' => true,
+            'SonataDoctrinePHPCRAdminBundle' => true,
+            'CmfSonataPhpcrAdminIntegrationBundle.' => true,
```

Comment:
i think the `.` at the end of CmfSonataPhpcrAdminIntegrationBundle. is not intentional, right? that indicates that we don't need this line - the bundle never checks if its loaded itself because if its not loaded it won't be executed.